libs/api/domains/driving-license/src/lib/graphql/models/qualityPhoto.model.ts

```diff
@@ -2,15 +2,12 @@ import { Field, ObjectType, ID } from '@nestjs/graphql'
 
 @ObjectType()
```

Comment:
Should also namespace the “ObjectType” in the schema. Think somehow like this:
```suggestion
@ObjectType(‘DrivingLicenseQualityPhoto’)
```